Dockerfile.rhel8

```diff
@@ -0,0 +1,26 @@
+# Build the manager binary
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10 AS builder
```

Comment:
question: about the 4.10 - as per our thread do we need to change for every release ?

Reply:
From  the info I've got , yes : every OCP release the dependencies and the Dockerfile should be updated.